### DISCUSSION TITLE
[Feature Request] Add "Tags" to custom notifications

### DISCUSSION THREAD

**Discussion by lonix1** [community/community#113612] [Repositories]
### Select Topic Area

Product Feedback

### Body

I use the `Watch -> Notifications -> Custom -> Releases` option so that I know when a repo releases a new version:
```
Notifications
  Issues
  Pull Requests
  Releases
  Discussions
  Security alerts
```

But many projects do not release binaries with a new version. For example, sometimes a new version is released as a package on an external site or package manager. In those cases, I won't be notified of the new version.

Please consider adding another custom notification event `Tags`. That will cover the cases where projects don't release binaries.

**Comment by Guiorgy** [comment]
Exactly, many projects have an alternative distribution method, for example Docker Hub. I don't understand why the option to be notified with new tags isn't a thing yet!
